silkworm/silkrpc/commands/erigon_api.cpp

```diff
@@ -272,15 +272,10 @@ awaitable<void> ErigonRpcApi::handle_erigon_watch_the_burn(const nlohmann::json&
             if (!cc) {
                 throw std::runtime_error("Invalid chain config");
             }
-            const auto block_reward{protocol::EthashRuleSet::compute_reward(*cc, block_with_hash->block)};
```

Comment:
I think we should leave the code as it is, and set to zero 
issuance.block_reward, issuance.ommer_reward, an issuance.issuance only if the protocol ruleset in the config is "Trust"

Reply:
After double check, this place should be fine as it's only setting rewards when ethash is used.